paddle/fluid/operators/benchmark/op_tester_config.h

```diff
@@ -0,0 +1,69 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#pragma once
+
+#include <istream>
+#include <string>
+#include <vector>
+#if !defined(_WIN32)
+#include <sys/time.h>
+#endif
+#include <chrono>  // NOLINT
+
+namespace paddle {
+namespace operators {
+namespace benchmark {
+
+struct Timer {
```

Comment:
Could directly use this ? 
https://github.com/PaddlePaddle/Paddle/blob/6402424f7a82c15ed361d76efcfbb46f79f873a4/paddle/fluid/platform/timer.h#L30

Reply:
Done.